packages/components/organisms/f-checkout/src/components/ErrorDialog.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+    <mega-modal
+        v-if="errorCode"
+        :is-open="isOpen"
+        data-test-id="checkout-issue-modal"
+        has-overlay
+        @close="handleClose">
+        <h3 data-test-id="checkout-issue-modal-title" class="u-noSpacing">
+            {{ $t(`errorMessages.checkoutIssues.${errorCode}.title`) }}
+        </h3>
+
+        <p data-test-id="checkout-issue-modal-message">
+            {{ $t(`errorMessages.checkoutIssues.${errorCode}.message`) }}
+        </p>
+
+        <f-button
+            :class="$style['c-checkout-redirectButton']"
+            button-type="primary"
+            button-size="large"
+            action-type="button"
+            data-test-id="redirect-to-menu-button"
+            @click.native="handleButtonClick">
```

Comment:
Should this be retrying the checkout call? Or is that coming later?

Reply:
I believe it's coming later as now we are just replicating iOS and Android.